color-bolts/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 25
```

Comment:
The `compileSdkVersion` should be in sync with the `BUILD_TOOLS_VERSION`, so better create a property for this one as well.

Reply:
Pushed, ready for check.

Reply:
Right, I noticed that myself later on.